proto/minder/v1/minder.proto

```diff
@@ -2559,70 +2559,73 @@ message ListEvaluationHistoryResponse {
 }
 
 message EvaluationHistory {
-    // entity contains details of the entity which was evaluated.
-    EvaluationHistoryEntity entity = 1;
+    message Entity {
+        // id is the ID of the entity.
+        string id = 1;
 
-    // rule contains details of the rule which the entity was evaluated against.
-    EvaluationHistoryRule rule = 2;
+        // type is the entity type.
+        minder.v1.Entity type = 2;
 
-    // status contains the evaluation status.
-    EvaluationHistoryStatus status = 3;
+        // name is the entity name.
+        string name = 3;
+    }
```

Comment:
Two comments:

1. I've rarely been happy in the future with where a nested message has been placed -- I'd lean towards making this a top-level message.
2. Can you include comments about uniqueness of the various fields, and ability to use those fields in future calls?  (For example, I assume that `id` is globally unique, but is `name` globally unique, or only unique within the scope of a Provider?)

@eleftherias on provider-scoping entity names, as I know she was looking to remove that linkage (and I was advocating for keeping it).